pkgs/by-name/vk/vkd3d-proton/sources.nix

```diff
@@ -5,11 +5,11 @@
 {
   vkd3d-proton = {
     pname = "vkd3d-proton";
-    version = "2.11.1-unstable-2024-03-07";
+    version = "2.12";
     src = fetchFromGitHub {
       owner = "HansKristian-Work";
       repo = "vkd3d-proton";
-      rev = "ffb04fb7091f932ca95da236d63ab9a4064d6443";
+      rev = "v2.12";
```

Comment:
```suggestion
      rev = "v${version}";
```

Reply:
I am still figuring out how to male this in a not-so-ugly way.